apps/backend/src/datasources/postgres/StatusActionsDataSource.ts

```diff
@@ -156,35 +182,63 @@ export default class PostgresStatusActionsDataSource
   }
 
   async addConnectionStatusActions(
-    input: AddConnectionStatusActionsInput
+    connectionStatusActionsInput: AddConnectionStatusActionsInput

```

Comment:
I can understand that there have been some refactoring on this function. Could you add that to the PR description for logging purpose.